app/src/js/main.js

```diff
@@ -4,11 +4,18 @@ petDetectrApp.formSetup = function formSetup() {
     // Grab radio buttons and text nodes
     this.petStatus = document.querySelectorAll('[name="status"]');
     this.lastSeenText = document.querySelectorAll('.js-seen-found-text');
-
+    this.isChippedSelector = document.querySelectorAll('[name=is-chipped]');
+    this.chipNumber = document.querySelector('#chip-number-wrap');
+    
     // add event listeners to radio buttons
     for (let i = 0, max = this.petStatus.length; i < max; i++) {
         this.petStatus[i].addEventListener('click', this.updateDateAndLocationText, true);
     }
+    
+    // event listener for chip number toggle
+    for (let i = 0; i < this.isChippedSelector.length; i++) {
+        this.isChippedSelector[i].addEventListener('click', this.hideShowChipNumber, true);
+    }
 };
```

Comment:
We're probably gonna have a few of these type of `for` loops.  I wonder if it's work abstracting to an `addEvent` or better named function where you pass in the array, event type and function? e.g 
```
function addEvent(array, eventType, func ) {
    for (let i = 0, max= this[ array ]length; i < max; i++) {
        this.[ array ][i].addEventListener(eventType, func, true);
    }
}

addEvent('click', this.isChippedSelector, this.hideShowChipNumber);
addEvent('click', this. updateDateAndLocationText, this.updateDateAndLocationText);

```